src/models/order.js

```diff
@@ -11,20 +11,20 @@ module.exports = (sequelize, DataTypes) => {
       // define association here
     }
   }
-  Orders.init(
+  Order.init(
     {
       orderNumber: DataTypes.INTEGER,
```

Comment:
This seems redundant to `id`. I was thinking that an entry for Order is only created when a user places it so we can replace this with `id`.